presto-docs/src/main/sphinx/connector/kinesis.rst

```diff
@@ -1,159 +1,195 @@
 =================
 Kinesis Connector
 =================
+
 Kinesis is Amazon's fully managed cloud-based service for real-time processing of large, distributed data streams.
 
 This connector allows the use of Kinesis streams as tables in Presto, such that each data-blob (message)
-in a kinesis stream is presented as a row in Presto.  A flexible table mapping approach lets us
+in a Kinesis stream is presented as a row in Presto. A flexible table mapping approach lets us
 treat fields of the messages as columns in the table.
 
-Under the hood, a Kinesis shard iterator is used to retrieve the records, along with a series of getRecords calls.
-The shard iterator starts by default 24 hours before the current time and works its way forward. To be able to query a stream, table mappings are needed. These table definitions can be
+Under the hood, a Kinesis
+`shard iterator <https://docs.aws.amazon.com/kinesis/latest/APIReference/API_GetShardIterator.html>`_
+is used to retrieve the records, along with a series of
+`GetRecords <https://docs.aws.amazon.com/kinesis/latest/APIReference/API_GetRecords.html>`_ calls.
+The shard iterator starts by default 24 hours before the current time and works its way forward.
+To be able to query a stream, table mappings are needed. These table definitions can be
 stored on Amazon S3 (preferred) or stored in a local directory on each Presto node.
 
-This connector is a read-only connector. It can only fetch data from kinesis streams, but cannot create streams or push
-data into existing streams.
+This connector is a read-only connector. It can only fetch data from Kinesis streams,
+but cannot create streams or push data into existing streams.
 
-To configure the Kinesis connector, create a catalog properties file, ``etc/catalog/CATALOG_NAME.properties`` with the following
-content and replace the properties as appropriate:
+To configure the Kinesis connector, create a catalog properties file ``etc/catalog/kinesis.properties``
```

Comment:
This should probably say “for example”. We don’t want to imply that the file *has* to be named exactly that.

Reply:
This is the language we use for other connectors. We explain that catalogs can be named anything in the main deployment documentation.